js/components/portal-dashboard/answers/answer-modal.tsx

```diff
@@ -34,7 +34,7 @@ export class AnswerModal extends PureComponent<IProps> {
           <div className={css.questionPrompt} data-cy="question-prompt">
             <QuestionIcon className={`${css.icon} ${css.questionTypeIcon}`} />
             <span className={css.questionPromptText} data-cy="question-prompt-text">
-              {prompt}
+              {prompt && prompt}
```

Comment:
This is odd.  Isn't prompt a string?  Shouldn't this just be `prompt` and then we determine above if it's empty string?

Reply:
I think we can safely just put in `{prompt}`.  Looks like in the worst case we just have an empty string "" that we will put in the span.

Reply:
I think we can safely just put in `{prompt}`.  In the worst case prompt will just have an empty string, I believe.